contracts/optimistic-ethereum/OVM/verification/OVM_StateTransitioner.sol

```diff
@@ -13,6 +13,7 @@ import { iOVM_StateTransitioner } from "../../iOVM/verification/iOVM_StateTransi
 import { iOVM_ExecutionManager } from "../../iOVM/execution/iOVM_ExecutionManager.sol";
```

Comment:
I think we also need to give some of the reward to the user who calls `executeTransaction`.